spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
I feel like this test is just testing rails, which is not necessary.  Was there another goal for the test that I'm missing?